lib/net.c

```diff
@@ -1038,6 +1043,29 @@ gatekeeper_get_rss_config(uint16_t port_id,
 	return ret;
 }
 
+int
+gatekeeper_setup_user(struct net_config *net_conf, const char *user)
+{
+	struct passwd *pw;
+
+	if (user == NULL) {
+		net_conf->pw_uid = getuid();
+		net_conf->pw_gid = getgid();
```

Comment:
Set `net_conf->pw_uid` and `net_conf->pw_gid` to zero instead of calling `getuid()` and `getgid()`.